src/microbe_stage/editor/CellEditorComponent.GUI.cs

```diff
@@ -370,6 +370,9 @@ private void CreateUndiscoveredOrganellesButtons(bool refresh = false, bool auto
 
         var worldAndPlayerArgs = GetUnlockPlayerDataSource();
 
+        // Apply LAWK settings so that no-unexpected organelles are shown
+        UpdateOrganelleLAWKSettings();
```

Comment:
If this change works, I think line 387 comment needs to be deleted so that there's no outdated comment left in the code.

Reply:
Upon playtesting I discovered that the below comment is indeed important, as the following steps lead to a bug:

- Start a new non-LAWK game
- Get to the editor
- Use the cheat to unlock all organelles
- Exit the editor and re-enter it
- Notice how the thermoplast organelle can now be placed even though LAWK is still on

So I think the code needs to be tweaked. I think easiest thing is to revert this change and instead modify the loop below to take LAWK into account and not adding something to the unlock conditions text when LAWK mode wouldn't allow it.